ros_ws/src/autonomous/wallfollowing2/script/wallfollowing.py

```diff
@@ -207,6 +211,53 @@ def laser_callback(scan_message):
     last_scan = scan_time
 
 
+def dyn_cfg_callback(config, level):
+    rospy.loginfo("""
+        Reconfigure Request:
+          min_throttle: {min_throttle},
+          max_throttle: {max_throttle},
+          radius_lower: {radius_lower},
+          radius_upper: {radius_upper},
+          steering_slow_down: {steering_slow_down},
+          steering_slow_down_dead_zone: {steering_slow_down_dead_zone},
+          max_acceleration: {max_acceleration},
+          corner_cutting: {corner_cutting},
+          straight_smoothing: {straight_smoothing},
+          barrier_size_realtive: {barrier_size_realtive},
+          barrier_size_realtive: {barrier_lower_limit},
+          barrier_upper_limit: {barrier_upper_limit},
+          barrier_exponent: {barrier_exponent},
+          controller_p: {controller_p},
+          controller_i: {controller_i},
+          controller_d: {controller_d}
```

Comment:
Consider implementing a `__str__` method for the parameters class. It could look like this:
```python
def __str__(self):
    return '\n'.join(name + ": " + str(getattr(self, name)) for name in self.names)
```

After creating the new parameters object, you can create the log message like this:
```python
rospy.loginfo('Reconfigure Request:\n' + str(parameters))
```